src/museums.ts

```diff
@@ -1,48 +1,91 @@
-import {
-  Curator,
-  Museum,
-  Origami,
-  utils,
-} from '@open-oracle-origami/origami-js-sdk'
+import { Curator, Museum, Origami } from '@open-oracle-origami/origami-js-sdk'
+import { ThirdwebSDK } from '@thirdweb-dev/sdk'
+import { TenetTestnet } from '@thirdweb-dev/chains'
 
 // TODO: this becomes it's own evm library or maybe it's an evm price aggregate library...
 // TODO: build threshold and deviation helpers
 // TODO: load from toml config
-const museumCollectionConfig = [
+const museumCollectionConfigs = [
   {
     id: 'tenet-testnet', // required
-    abi: [], // this can default to origami-solidity if not passed
-    seedPhrase: 'hello world', // required unless privateKey
```

Comment:
You'll still need the seedPhrase and or privateKey, these can change per museum and per collection.  In theory every collection could have their own owner account. We can name it to seedPhraseEnvKey or something like that and pull it from the env on demand.